Add migrations for table creation to role-service

tayareut opened the pull request:
- Add changelog files
- Add liquibase.properties
- Update pom.xml
- Update application.properties

tayareut committed:
Add migrations for table creation to role-service
- Add changelog files
- Add liquibase.properties
- Update pom.xml
- Update application.properties

keep-it-unreal commented on pom.xml:
```diff
@@ -50,6 +59,26 @@
 
 	<build>
 		<plugins>
```
может быть использовать spring-boot-maven-plugin плагин, а dependencies оставить только те, что указаны выше?) 
               <plugins>
			<plugin>
				<groupId>org.springframework.boot</groupId>
				<artifactId>spring-boot-maven-plugin</artifactId>
				<version>${project.parent.version}</version>
			</plugin>
		</plugins>

tayareut replied:
он есть ниже, или он может заменить полностью liquibase-maven-plugin?

keep-it-unreal replied:
Не уверен, но кажется, что должно. Читал инструкцию по использованию, там предлагается использовать spring-boot-maven-plugin https://contribute.liquibase.com/extensions-integrations/directory/integration-docs/springboot/using-springboot-with-maven/

tayareut committed:
Add migrations for table creation to role-service & Update docker-compose to create a local db when started
- Update changeset files
- Update pom.xml
- Update application.yml
- Update docker-compose.yml
- Update README.md

tayareut committed:
Disable the security auto-configuration
- Disable the Auto-Configuration in RoleServiceApplication.java
- Update pom.xml - update spring-boot-starter-parent version to fix some issues with spring-boot-starter-security
- Update application.yml
- Update docker-compose.yml
- Delete: application.properties
          liquibase.properties
          01-create-users-table-changeset.xml
          02-create-roles-table-changeset.xml
          05-create-users-roles-table-changeset.xml
